en-US/navigation.lang

```diff
@@ -303,3 +309,183 @@ Get a Firefox Account
 
 ;Check out the Benefits
 Check out the Benefits
+
+
+;Firefox Blog
+Firefox Blog
+
+
+;Read about new Firefox features, and get tips for staying safer online.
+Read about new Firefox features, and get tips for staying safer online.
+
+
+;Meet the technology company that puts people before profit.
+Meet the technology company that puts people before profit.
+
+
+;The non-profit behind Firefox is fighting for a healthy internet for all.
+The non-profit behind Firefox is fighting for a healthy internet for all.
+
+
+;Learn how Firefox treats your data with respect.
+Learn how Firefox treats your data with respect.
+
+
+;Mozilla Careers
+Mozilla Careers
+
+
+;Approach your career with a sense of purpose. Find worthy work at Mozilla.
+Approach your career with a sense of purpose. Find worthy work at Mozilla.
+
+
+;Browsers
+Browsers
+
+
+# Accessible label for the 'X' button that closes a menu
+;Close Browsers menu
+Close Browsers menu
+
+
+;Get the browser that respects your privacy automatically. On Windows, MacOS or Linux.
+Get the browser that respects your privacy automatically. On Windows, MacOS or Linux.
+
+
+;Features
+Features
+
+
+;Release Notes
+Release Notes
+
+
+;Extensions
+Extensions
+
+
+;Firefox Desktop Browser
+Firefox Desktop Browser
+
+
+;Firefox Mobile Browsers
+Firefox Mobile Browsers
+
+
+;Take speed, privacy and peace of mind with you. On Android and iOS.
+Take speed, privacy and peace of mind with you. On Android and iOS.
+
+
+;Firefox for Enterprise
+Firefox for Enterprise
+
+
+;Get unmatched data protection on the release cadence that suits your organization.
+Get unmatched data protection on the release cadence that suits your organization.
+
+
+;Firefox Reality
```

Comment:
If this is the product name, which means it is left in English. Then we can remove this string.